draft-ietf-quic-transport.md

```diff
@@ -606,13 +614,40 @@ packet containing the initial cryptographic message needs to be created, this
 includes the packets sent after receiving a Version Negotiation
 ({{packet-version}}) or Retry packet ({{packet-retry}}).
 
+When a server receives an Initial packet with an address validation token, it
+should attempt to validate it.  If the token is invalid then it should just be
+ignored.  If the validation succeeds, the server should then allow future
```

Comment:
There's something I find confusing about "future connection handshakes to proceed".  Doesn't validation allow the current handshake to proceed?